ruby/ruby/Glacier2.rb

```diff
@@ -2,6 +2,6 @@
 # Copyright (c) ZeroC, Inc. All rights reserved.
 #
 
-require 'Glacier2/Router'
-require 'Glacier2/PermissionsVerifier'
-require 'Glacier2/Metrics'
+require_relative 'Glacier2/Router'
```

Comment:
We should consider _not_ shipping this generated code. That is, if a user wants to create a Ruby client to Glacier2/IceGrid/IceStorm, this user can just compile the corresponding Slice files. Not for this PR.